src/Servers/test/FunctionalTests/ServerComparison.FunctionalTests.csproj

```diff
@@ -5,6 +5,7 @@
 
     <!-- Tests do not work on Helix or when bin/ directory is not in project directory due to undeclared dependency on test content. -->
     <BuildHelixPayload>false</BuildHelixPayload>
+    <SkipTests>false</SkipTests>
```

Comment:
Can't you just query BuildHelixPayload?

Reply:
Possibly, what are you suggesting?

Reply:
Is it possible to define skip tests globally based on BuildHelixPayload?

Reply:
I'm not a msbuild pro at all, my naive understanding is that its just evaluating files in order, so there's no way to default it dynamically globally so when its turned off later in a particular project it just works, which is why I just went with blasting it both flags to false.  There's no way to defer evaluation by reference like that, I don't think.

Reply:
@HaoK suggest you move the setting to the root Directory.Build.targets file (after line 110 or so, where `$(BuildHelixPayload)` gets its default) and change the condition to
``` msbuild
<SkipTests Condition="'$(SkipHelixReadyTests)' == 'true' AND '$(BuildHelixPayload)' == 'true'">true</SkipTests>
```

Reply:
awesome this is much better!